deployment/updates/scripts/xml/notifications/2019_01_15_update_unique_mediaspace_notifications_templates.template.xml

```diff
@@ -0,0 +1,586 @@
+    <!-- User was removed from category -->
+    <multirequest>
+        <!-- Get notification template using system name -->
+        <request service="eventNotificationTemplate" action="listTemplates" plugin="eventNotification" partnerId="0">
+            <filter objectType="KalturaEventNotificationTemplateFilter">
+                <systemNameEqual>Unique_Kms_User_Was_Removed_From_Category</systemNameEqual>
+            </filter>
+            <pager objectType="KalturaFilterPager">
+                <pageSize>1</pageSize>
+            </pager>
+        </request>
+        <!-- Update notification template -->
+        <request service="eventNotificationTemplate" action="update" plugin="eventNotification" partnerId="0">
+            <id>{1:result:objects:0:id}</id>
+            <template objectType="KalturaEmailNotificationTemplate">
+                <name>User was removed from category (Application Specific)</name>
+                <body>You have been removed from {category_name} by {user_id}</body>
+                <contentParameters objectType="array">
+                    <!-- Existing parameters -->
+                    <item objectType="KalturaEventNotificationParameter">
+                        <key>from_email</key>
+                        <description>Server configuration: partner_notification_email</description>
+                        <value objectType="KalturaEvalStringField">
+                            <code>kConf::get("partner_notification_email")</code>
+                        </value>
+                    </item>
+                    <item objectType="KalturaEventNotificationParameter">
+                        <key>from_name</key>
+                        <description>Server configuration: partner_notification_name</description>
+                        <value objectType="KalturaEvalStringField">
+                            <code>kConf::get("partner_notification_name")</code>
+                        </value>
+                    </item>
+                    <item objectType="KalturaEventNotificationParameter">
+                        <key>category_name</key>
+                        <description>Category name</description>
+                        <value objectType="KalturaEvalStringField">
+                            <code>($category = $scope->getObject()->getcategory()) ? $category->getName() : ''</code>
+                        </value>
+                    </item>
+                    <!-- New parameters -->
+                    <item objectType="KalturaEventNotificationParameter">
+                        <key>category_id</key>
+                        <description>Category ID</description>
+                        <value objectType="KalturaEvalStringField">
+                            <code>$scope->getObject()->getCategoryId()</code>
+                        </value>
+                    </item>
+                    <item objectType="KalturaEventNotificationParameter">
+                        <key>user_id</key>
```

Comment:
@yusitnikov I'd name the parameter something like 'executing_user_id' or 'ks_user_id'.

Reply:
Done.